src/device.c

```diff
@@ -269,6 +269,17 @@ void dpdk_send_all_packets(uint8_t port_id, uint16_t queue_id, struct rte_mbuf**
 	return;
 }
 
+void send_single_packet(uint8_t port_id, uint16_t queue_id, struct rte_mbuf* pkt) {
```

Comment:
please rename this to phobos_* to be consistent with the other send/rx functions